src/model/createModel.ts

```diff
@@ -51,7 +53,10 @@ export function createModel<
       const propertyDefinition = get(definition, propertyName)
 
       // Ignore relational properties at this stage.
-      if (propertyDefinition instanceof Relation) {
+      if (
+        propertyDefinition instanceof Relation ||
+        propertyDefinition instanceof NullableRelation
```

Comment:
Shall we consider implementing `NullableRelation` as a child of the `Relation` base class?

That way we don't have to have any explicit `instanceof NullableRelation` checks.

```js
import { Relation } from './relation'

export class NullableRelation extends Relation {
  // Add nullable values support
}

const relation = new NullableRelation()
if (relation instanceof Relation) {
  // still true
}
```

Reply:
With inheritance, we can utilize parent class checks when we intend to handle `Relation` and `NullableRelation` the same, and strict `instanceof NullableRelation` when we wish to implement the logic specific only to the nullable relation. 

What do you think about this?